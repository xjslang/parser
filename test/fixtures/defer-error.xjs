function testWithError() {
  console.log('start')
  defer {
    console.log('cleanup executed')
  }
  
  try {
    throw new Error('test error')
  } catch (e) {
    console.log('error caught')
  }
}
testWithError()
