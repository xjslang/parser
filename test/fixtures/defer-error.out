start
error caught
cleanup executed
